decidim-admin/app/mailers/decidim/admin/application_mailer.rb

```diff
@@ -5,8 +5,19 @@ module Admin
     # Custom application mailer, scoped to the admin mailer.
     #
     class ApplicationMailer < ActionMailer::Base
```

Comment:
```suggestion
    class ApplicationMailer < ::Decidim::ApplicationMailer
```

Reply:
I agree, it doesn't make much sense this mailer. I'll review it

Reply:
I've removed it as I couldn't find any class inheriting from `Decidim::Admin::ApplicationMailer` neither. It's done in [085a0ce](https://github.com/decidim/decidim/pull/12837/commits/085a0ce0583ac67952fb3523a82e8cf0df21633a)